src/glusterfs_lib/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ add_library(
         xglfs_ftruncate.c
         xglfs_utimens.c
         xglfs_statfs.c
+        xglfs_readdir.c
```

Comment:
how come we didn't have it before?

Reply:
reading functions of glusterfs wasn't necessary before